bgpd/bgp_vty.c

```diff
@@ -6159,21 +6159,46 @@ static int set_ecom_list(struct vty *vty, int argc, struct cmd_token **argv,
 	return CMD_SUCCESS;
 }
 
-static int vpn_policy_getafi(struct vty *vty, int *doafi)
+/*
+ * v2vimport is true if we are handling a `import vrf ...` command
+ */
+static afi_t vpn_policy_getafi(struct vty *vty, struct bgp *bgp, bool v2vimport)
 {
+	afi_t afi;
+
 	switch (vty->node) {
 	case BGP_IPV4_NODE:
-		doafi[AFI_IP] = 1;
+		afi = AFI_IP;
 		break;
 	case BGP_IPV6_NODE:
-		doafi[AFI_IP6] = 1;
+		afi = AFI_IP6;
 		break;
 	default:
 		vty_out(vty,
 			"%% context error: valid only in address-family <ipv4|ipv6> unicast block\n");
-		return CMD_WARNING_CONFIG_FAILED;
+		return AFI_MAX;
```

Comment:
I agree with the changes to a single afi return value (previous multi-value was an artifact of prior CLI syntax).

Reply:
ok good!